web/ui/react-app/src/pages/targets/SeriesLabels.test.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint @typescript-eslint/camelcase: 0 */
```

Comment:
I assume this is for `__metrics_path__`. Would quoting that object key (`"__metrics_path__": ...`) avoid the linter problem without turning that check off completely?

Reply:
Unfortunately, it does not. I can disable eslint for the line if you prefer

Reply:
Huh weird, ok. Meh, since it's a test, I think just keep it.